### ISSUE TITLE
Dropdown menu items cannot be right-clicked

### ISSUE THREAD

**Issue by lindapaiste** [processing/p5.js-web-editor#2359]
### Increasing Access

Allows users to open a link in a new tab, etc.

### Feature enhancement details

I wanted to open "My Collections" in a new tab, but the menu closed immediately when I right-clicked the link.  Opening in a new tab is possible via `Ctrl` + click.  But right-click should work too.
![image](https://github.com/processing/p5.js-web-editor/assets/28965286/674077f6-505e-4616-974e-b393c21443eb)

**Comment by Biki-das** [comment]
@lindapaiste could you clarify the behaviour you are expecting, do you want the drop item not to disappear after selecting item from it? or you want to open the link in a new tab Everytime there is a right click?

**Comment by Biki-das** [comment]
https://github.com/processing/p5.js-web-editor/assets/72331432/81528c00-0455-441c-bcbe-9b5bb79a7c3f

i think i understood the problem correctly now, the `mouseup` event makes the `setDropdownOpen` to none resulting in the drop down closed, i have added a filter to look for right click button and just doing an early return. this is the behaviour on right click currently. also work with ctrl + right click

shall raise the PR!

**Comment by sdivyanshu90** [comment]
Hello @lindapaiste,
I want to clarify that I've been actively addressing this issue, as I mentioned in my previous comment. Additionally, I think it is a good practice to check whether someone is already working on it before making any submission or PR.

**Comment by Biki-das** [comment]
> Hello @lindapaiste, I want to clarify that I've been actively addressing this issue, as I mentioned in my previous comment. Additionally, I think it is a good practice to check whether someone is already working on it before making any submission or PR.

the issue was not officially assigned so anyone who solves it early solves it
